src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/enable-aws-lambda-monitoring.mdx

```diff
@@ -1,10 +1,6 @@
 ---
 title: Enable monitoring for AWS Lambda
```

Comment:
@zuluecho9, the number of steps in this document don't match the number of steps in the left-navigation pane. Can you think of a way to harmonize these?